web/packages/core/src/polyfills.ts

```diff
@@ -68,14 +68,13 @@ function polyfillFrames(): void {
     iframes = iframes ?? document.getElementsByTagName("iframe");
     frames = frames ?? document.getElementsByTagName("frame");
 
-    [iframes, frames].forEach((elementsList) => {
-        for (let i = 0; i < elementsList.length; i++) {
-            const element = elementsList[i];
-            if (element.dataset.rufflePolyfilled !== undefined) {
+    [iframes, frames].forEach((elements) => {
+        for (const element of elements) {
+            if (element.dataset["rufflePolyfilled"] !== undefined) {
```

Comment:
I thought `dataset.rufflePolyfilled` is the more idiomatic way of writing this?

Reply:
`@tsconfig/strictest` turns on [`noPropertyAccessFromIndexSignature`](https://www.typescriptlang.org/tsconfig#noPropertyAccessFromIndexSignature), which bans "indexed" properties from being accessed via the dot operator.